tests/loggers/test_all.py

```diff
@@ -365,6 +365,16 @@ def _test_logger_created_on_rank_zero_only(tmpdir, logger_class):
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
+def test_none_logger(tmpdir):
+    """Tests the case where logger=None is passed to Trainer."""
+    model = BoringModel()
+    trainer = Trainer(
+        logger=None,
+        max_steps=1,
+    )
+    trainer.fit(model)
```

Comment:
You don't need to call trainer.fit 

Directly assert that trainer.logger, trainer.loggers, and trainer.log_dir return what's expected. This test case can cover logger=None, False, and []

Reply:
Updated!